src/plugins/lists/entry/hooks.tsx

```diff
@@ -28,10 +28,12 @@ export const EntryListContainer = createPluginContainer(() => {
         {
           label: 'Entry Title',
           name: 'title',
+          type: 'text',
```

Comment:
Can remove all the places where you added type: 'text' by making it optional as it is above.

Reply:
Done